README.md

```diff
@@ -1808,6 +1809,33 @@ Produces a duplicate-free version of the array, using === to test object equalit
 
 **[⬆ back to top](#quick-links)**
 
+### _.uniqBy
+
+Like `_.uniq` but compares the elements using the provided function or object key.
+
+  ```js
+  // Lodash: _.uniqBy
+  // Underscore: _.uniq
+  _.uniqBy([2.1, 1.2, 2.3], Math.floor);
+  // => [2.1, 1.2]
+  _.uniqBy([{ 'x': 1 }, { 'x': 2 }, { 'x': 1 }], 'x');
+  // => [{ 'x': 1 }, { 'x': 2 }]
+
+  // Native
+  [...new Map([2.1, 1.2, 2.3].map(v => [Math.floor(v), v]).reverse()).values()];
```

Comment:
To maintain the convention, you should add the function definition here along with an example of its invocation.